src/seedu/addressbook/AddressBook.java

```diff
@@ -442,6 +474,12 @@ private static String getMessageForSuccessfulAddPerson(String[] addedPerson) {
                 getNameFromPerson(addedPerson), getPhoneFromPerson(addedPerson), getEmailFromPerson(addedPerson));
     }
 
+
+    private static String getMessageForSuccessfulEditPerson(String[] newPersonRecord) {
```

Comment:
Missing header comment. All non trivial methods should have javadoc format header comments.